app/actions/MomentActionCreators.js

```diff
@@ -0,0 +1,37 @@
+import { MOMENT_ACTIONS } from '../constants';
+import { likeMomentAPI } from '../api/MomentAPI';
+
+
+const momentActionSuccess = (actionType, payload) => ({
+  type: MOMENT_ACTIONS[`${actionType}Success`], success: true, payload 
+});
+
+const momentActionError = (actionType, error) => ({
+  type: MOMENT_ACTIONS[`${actionType}Error`], success: false, error 
+});
+
+const triggerlikeMoment = () => ({
+  type: MOMENT_ACTIONS.likeRequest
+});
+
+export const buildRequestParams = (jsonParams) => {
+  const form_data = new FormData();
+  form_data.append('data', jsonParams);
```

Comment:
I think you should do the `JSON.stringify` here instead of in `likeMoment`. stringify at the last moment